README.md

```diff
@@ -192,6 +194,19 @@ python prompt.py -e {experiment_name}
 where `experiment_name` is the output folder of the experiment you want to chat with (see configuration).
 The interactive chat will also work with model that were finetuned using the UI.
 
+To publish the model to Hugging Face, use the following command:
+
+```bash
+python publish_to_hugging_face.py -e {experiment_name} -d {device} -a {api_key} -u {user_id} -m {model_name}
+``` 
+
+`experiment_name` is the output folder of the experiment.
+`device` is the target device for running the model, either 'cpu' or 'cuda:0'.
+`api_key` is the Hugging Face API Key. If user logged in, it can be omitted.
+`user_id` is the Hugging Face user ID. If user logged in, it can be omitted.
+`model_name` is the name of the model to be published on Hugging Face.
+

```

Comment:
add safetensor feature

Reply:
`safe_serialization` is a flag indicating whether safe serialization should be used. Default is True.

Done